### PR TITLE
Write documentation for Custom Schema Plugin

### PR DESCRIPTION
**Pull request by hand-dot**
ref: https://github.com/pdfme/pdfme/issues/213

### CODE REVIEW

File: website/docs/custom-fonts.md [hunk 1]
```diff
@@ -38,17 +52,18 @@ Let's check out how to set font in the generator and ui packages.
 Set font as option in [generate](/docs/getting-started#generator) function
 
 ```ts
-import { Template, BLANK_PDF, generate } from '@pdfme/generator';
+import { Template, BLANK_PDF, Font } from '@pdfme/common';
```
Comment (peteward) [inline website/docs/custom-fonts.md:55]: import type?

**Review by peteward — COMMENTED** [review]
Apologies for the late review and amazing work on updating all of this documentation.

I was trying to see how to update my app and I found a few inconsistencies in the docs so I thought I should read through the whole PR.

Many of my comments about `import` vs `import type` can probably be ignored.
I'm guessing that if you are importing more than 1 thing from a package then you maybe don't have to use the `type` prefix?

**Comment by hand-dot** [comment]
@peteward Thank you! I will correct it based on your comment.

It would be a pleasure to collaborate with you.

File: website/docs/development-guide.md [hunk 2]
```diff
@@ -29,19 +30,22 @@ As of writing this page, we just released Version 2 two weeks ago. However, we a
...
     - Currently built-in schemas will be packaged separately, and improvements will be made individually.
-  - Custom Design for UI
+  - [Custom Design for UI](https://github.com/pdfme/pdfme/issues/243)
     - Allow changes to the accent color (currently it's light blue)
     - Add class names to each UI component to allow customization from the outside.
+  - [Custom Label for UI](https://github.com/pdfme/pdfme/issues/107)
+    - Allow users to change the label of the UI component from the outside.
+    - Overwrite the any language label with the user's own label
 
 ## Need technical support?
```
Comment (peteward) [inline website/docs/development-guide.md:51]: maybe update this for Discord?

File: README.md [hunk 3]
```diff
@@ -12,7 +13,7 @@
   <a href="https://npmcharts.com/compare/@pdfme/common?minimal=true">
     <img src="https://img.shields.io/npm/dm/@pdfme/common.svg" alt="Downloads per month on npm." />
   </a>
-  <a href="https://pdfme.com/development-guide#contribution">
+  <a href="https://pdfme.com/docs/development-guide#contribution">
```
[the remark this answers is not in the document]
Comment (hand-dot) [reply README.md:16]: The URL of the website has been partially modified. (I've mainly placed it under the "docs" directory.) 

Therefore, this URL will work correctly once the website is released.

File: README.md [hunk 4]
```diff
@@ -74,7 +75,7 @@ The packages for using PDF designer, forms and viewers can be installed with the
 npm i @pdfme/ui @pdfme/common
 ```
 
-*You must install `@pdfme/common` regardless of which package you use.
+\*You must install `@pdfme/common` regardless of which package you use.
```
[the remark this answers is not in the document]
Comment (hand-dot) [reply README.md:78]: > was this intentional?
Yes

File: website/docs/development-guide.md [hunk 5]
```diff
@@ -29,19 +30,22 @@ As of writing this page, we just released Version 2 two weeks ago. However, we a
 
 As we aim for the release of the next version, given our limited resources, we anticipate it being a long-term plan. Therefore, to ensure we do not lose sight of our developmental direction and to address our users who are anticipating new features, we have written this document.
 
-- ### [V2 Roadmap](https://github.com/orgs/pdfme/projects/5/views/1)  
-In this version, we mainly address what needs to be done before the release of V3. Development in V2 will not involve any schema other than the text type schema, and we will proceed with development that doesn't depend on schemas. While being conscious of the V3 release, we will improve the completeness of pdfme itself and work towards decoupling schemas from pdfme.
+- ### [V2 Roadmap](https://github.com/orgs/pdfme/projects/5/views/1)
```
[the remark this answers is not in the document]
Comment (hand-dot) [reply website/docs/development-guide.md:33]: I'll remove this V2 Roadmap item

File: packages/generator/__tests__/generate.test.ts [hunk 6]
```diff
@@ -3,7 +3,7 @@ import * as path from 'path';
 import generate from '../src/generate';
 import templateData from './assets/templates';
 import { Template, Font, BLANK_PDF, Schema } from '@pdfme/common';
```
[the remark this answers is not in the document]
Comment (hand-dot) [reply packages/generator/__tests__/generate.test.ts:5]: I'm importing BLANK_PDF along with it, so I'm not using `import type`. 

I don't have a particular preference, but we do use `import type` when importing only type information.

File: packages/schemas/src/text/uiRender.ts [hunk 7]
```diff
@@ -30,6 +30,16 @@ const mapVerticalAlignToFlex = (verticalAlignmentValue: string | undefined) => {
   return 'flex-start';
 };
 
+function getBackgroundColor(mode: 'form' | 'viewer' | 'designer', value: string, schema: Schema) {
```
[the remark this answers is not in the document]
Comment (hand-dot) [reply packages/schemas/src/text/uiRender.ts:33]: Oh, I'll change arrow func.

File: website/docs/development-guide.md (hunk 2 above)
Comment (hand-dot) [reply website/docs/development-guide.md:51]: @peteward 
You're right.

However, I'm concerned about the burden on Peter. You wouldn't want to offer support for free, would you?

I'd like to hear your thoughts.

**Pull request merged**